### PR TITLE
Added some documentation!

### PR DESCRIPTION
**Pull request by arm61**
you're welcome :)

### CODE REVIEW

File: pixel_art.py [hunk 1]
```diff
@@ -1,4 +1,16 @@
+"""
+A simple pixel art script, currently including 
+the ability to draw:
+- a bee
+"""
+
 def define_bee():
+    """
+    Draws a pctiure of a bee using a pixal art design.
```
Comment (arm61) [inline pixel_art.py:9]:
```suggestion
    Draws a picture of a bee using a pixal art design.
```

**Review by arm61 — COMMENTED** [review]
Typos! Lazy work

File: pixel_art.py (hunk 1 above)
Comment (maximillian-dolan) [reply pixel_art.py:9]: well corrected

Comment (rahmcj) [inline pixel_art.py:9]:
```suggestion
    Draws a picture of a bee using a pixel art design.
```

File: pixel_art.py [hunk 2]
```diff
@@ -24,6 +36,12 @@ def define_bee():
 
 
 def plot_image(image):
+    """
+    Plots the image as a simple mpl pllt.
```
Comment (alex-naz) [inline pixel_art.py:40]: Why "pllt" ?

File: pixel_art.py [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+A simple pixel art script, currently including 
+the ability to draw:
+- a bee
+"""
+
 def define_bee():
+    """
+    Draws a pctiure of a bee using a pixal art design.
+
+    :returns: Array of RBG values.
```
Comment (KayVeeZ) [inline pixel_art.py:11]:
```suggestion
    :returns: Array of RBG values.
    # kayveez was here :)
```

**Review by KayVeeZ — COMMENTED** [review]
github is awesome

**Pull request closed without merging**